src/bus.rs

```diff
@@ -138,12 +143,18 @@ impl<PINS: Send+Sync> usb_device::bus::UsbBus for UsbBus<PINS> {
             delay(72);
 
             regs.btable.modify(|_, w| w.btable().bits(0));
-            regs.cntr.modify(|_, w| w
-                .fres().clear_bit()
-                .resetm().set_bit()
-                .suspm().set_bit()
-                .wkupm().set_bit()
-                .ctrm().set_bit());
+            regs.cntr.modify(|_, w| {
+                w.fres()
+                    .clear_bit()
+                    .resetm()
+                    .set_bit()
+                    .suspm()
+                    .set_bit()
+                    .wkupm()
+                    .set_bit()
+                    .ctrm()
+                    .set_bit()
```

Comment:
This "common format style" is questionable.

Reply:
Almost rust librarie include embedded_hal formated by rust-fmt because it common for everythere. No more objective than - one style for everything.